api/logs.go

```diff
@@ -139,6 +152,7 @@ func (api *LogsAPI) GetLogs(ctx context.Context, crit FilterQuery) ([]*logType,
 		}
 	}()
 	rows, err := api.db.QueryContext(ctx, query, params...)
+	log.Debug("params", "params", params)
```

Comment:
If this still needs to be here, let's make the message more descriptive. (At least it's debug level though)